src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -124,6 +124,10 @@ void CoilCoolingDXCurveFitPerformance::instantiateFromInputSpec(EnergyPlus::Ener
         setOperMode(state, this->alternateMode2, 3);
     }
 
+    if (!input_data.outdoor_temperature_dependent_crankcase_heater_capacity_curve_name.empty()) {
+        this->crankcaseHeaterCapacityCurveIndex =
+            Curve::GetCurveIndex(state, input_data.outdoor_temperature_dependent_crankcase_heater_capacity_curve_name);
+    }
```

Comment:
The input processing needs more steps here (and everywhere) to check if a valid curve was found and to check the curve dimensions. For example, see [here](https://github.com/NREL/EnergyPlus/blob/557201b3c0ec9d8b5e7235cd4c8d893c4e01fef4/src/EnergyPlus/DXCoils.cc#L1320-L1335). For this application, any uni-variate curve or table is acceptable.

Reply:
good point. I'll add in the curve validations.